### DISCUSSION TITLE
Correct way to set keybindings on macOS (without sound)

### DISCUSSION THREAD

**Discussion by julien** [ghostty-org/ghostty#3711] [Q&A] [answered]
I have defined these two keybidings on macOS

```
keybind = ctrl+cmd+right=text:\x1b[1;5C
keybind = ctrl+cmd+left=text:\x1b[1;5D
```

This might seem wrong, but it works; however when pressing those keys (ctrl+cmd+left or ctrl+cmd+right), macOS also plays an "alert sound" and I wanted to know if there's a way to disable this.

**Comment by taylrfnt** [comment, marked as answer]
Just mentioning #5521 here as related (and also a new key combo triggers the console beep)
